Automattic/simplenote-electron
# Adding 'unplated' icons for .appx builds

During the transition to using server builds for .appx packages we missed adding the `unplated` icons, which tells Windows that we don't want the square background to be used for the app icon.

Before (forgive the pixelated look, just my VM):
![screen shot 2018-12-03 at 3 22 58 pm](https://user-images.githubusercontent.com/789137/49410400-49406480-f719-11e8-90f9-0ebf4b2862fc.png)

After:
![screen shot 2018-12-03 at 4 32 40 pm](https://user-images.githubusercontent.com/789137/49410407-4d6c8200-f719-11e8-96c1-bd903c34adcd.png)

We also had a squashed `StoreIcon`, which should have been 50x50 which I've corrected.

**To Test**
Install the .appx built by appveyor for this PR. You should see the unplated icon in the task bar.
